app/screens/phone-auth-screen/phone-input.tsx

```diff
@@ -1,396 +1,216 @@
-              textContainerStyle={styles.textContainer}
-              defaultValue={phoneNumber}
-              defaultCode={defaultCode}
-              layout="first"
-              renderDropdownImage={
-                <DownArrow testID="DropDownButton" width={12} height={14} />
+        <View style={styles.textContainer}>
+          <Text type={"p1"}>
+            {showCaptcha
+              ? LL.PhoneInputScreen.headerVerify()
+              : LL.PhoneInputScreen.header()}
+          </Text>
+        </View>
+
+        <View style={styles.phoneEntryContainer}>
+          <CountryPicker
+            countryCode={
+              (phoneInputInfo?.countryCode || DEFAULT_COUNTRY_CODE) as CountryCode
+            }
+            countryCodes={getCountries() as CountryCode[]}
+            onSelect={(country) => setCountryCode(country.cca2 as PhoneNumberCountryCode)}
+            containerButtonStyle={styles.countryPickerButtonStyle}
+            withCallingCodeButton={true}
+            withFilter={true}
+            withCallingCode={true}
+          />
+          <View style={styles.numberContainer}>
+            <Text
+              type="h2"
+              color={
+                phoneInputInfo?.formattedPhoneNumber ? undefined : theme.colors.grey8
               }
-              textInputProps={{
-                placeholder: LL.PhoneInputScreen.placeholder(),
-                returnKeyType: loadingRequestPhoneCode ? "default" : "done",
-                onSubmitEditing: Keyboard.dismiss,
-                keyboardType: "phone-pad",
-                textContentType: "telephoneNumber",
-                accessibilityLabel: "Input phone number",
-              }}
-              countryPickerProps={{
-                modalProps: {
-                  testID: "country-picker",
-                },
-              }}
-              codeTextStyle={styles.codeTextStyle}
-              autoFocus={true}
-            />
-            <ActivityIndicator
-              animating={loadingRequestPhoneCode}
-              size="large"
-              color={color.primary}
-              style={styles.activityIndicator}
-            />
-            <View style={styles.buttonsContainer}>
-              <Button
-                containerStyle={styles.button}
-                titleStyle={styles.buttonTitle}
-                title={LL.PhoneInputScreen.sms()}
-                disabled={Boolean(phoneNumber)}
-                onPress={submitViaSms}
-              />
-              <GaloyTertiaryButton
-                containerStyle={styles.button}
-                title={LL.PhoneInputScreen.whatsapp() + "  "}
-                onPress={submitViaWhatsapp}
-                icon={<Icon name="logo-whatsapp" size={14} color={color.primary} />}
-              />
-              <ContactSupportButton containerStyle={styles.button} />
-            </View>
-          </KeyboardAvoidingView>
+            >
+              {phoneInputInfo?.formattedPhoneNumber || DEFAULT_PHONE_NUMBER}
```

Comment:
it seem we can't paste a phone number any more

Reply:
Just to clarify you want to be able to paste your phone number in the phone input? What is the scenario where someone has first copied their phone number and then pastes it?